scripts/temp-create-data.js

```diff
@@ -1,56 +1,82 @@
-const { utils, Wallet, Contract, ContractFactory, providers } = require('ethers');
-
-const colonyJSExtras = require('@colony/colony-js/extras')
-const colonyJSIColony = require('../node_modules/@colony/colony-js/dist/cjs/contracts/IColony/9/factories/IColony__factory.js')
-const { addAugmentsB } = require('../node_modules/@colony/colony-js/dist/cjs/clients/Core/augments/AddDomain.js');
+const {
+  utils,
+  Wallet,
+  Contract,
+  ContractFactory,
+  providers,
+} = require("ethers");
+
+const colonyJSExtras = require("@colony/colony-js/extras");
+const colonyJSIColony = require("../node_modules/@colony/colony-js/dist/cjs/contracts/IColony/9/factories/IColony__factory.js");
+const {
+  addAugmentsB,
+} = require("../node_modules/@colony/colony-js/dist/cjs/clients/Core/augments/AddDomain.js");
 
 /*
  * @NOTE To preserve time, I just re-used a script I wrote for one of the lambda functions
  * So if that lambda function gets removed, this script will stop working
  */
-const { graphqlRequest } = require('../amplify/backend/function/createUniqueColony/src/utils');
+const {
+  graphqlRequest,
+} = require("../amplify/backend/function/createUniqueColony/src/utils");
 /*
  * @NOTE This script depends on both the ganache chain (and especially accounts) being active
  * as well as the network contracts being deployed on said chain
  * So make sure to only run this script after the dev environment (via docker compose) was started
  */
-const { private_keys } = require('../amplify/mock-data/colonyNetworkArtifacts/ganache-accounts.json');
-const { etherRouterAddress } = require('../amplify/mock-data/colonyNetworkArtifacts/etherrouter-address.json')
+const {
+  private_keys,
+} = require("../amplify/mock-data/colonyNetworkArtifacts/ganache-accounts.json");
+const {
+  etherRouterAddress,
+} = require("../amplify/mock-data/colonyNetworkArtifacts/etherrouter-address.json");
 
-const API_KEY = 'da2-fakeApiId123456';
-const GRAPHQL_URI = 'http://localhost:20002/graphql';
+const API_KEY = "da2-fakeApiId123456";
+const GRAPHQL_URI = "http://localhost:20002/graphql";
```

Comment:
I wanted to change the usernames to be different from the Colony names to prevent any misreading of the data. Other changes were collateral damage, with ESLint/prettier not running on JS files.